bin/pycbc_upload_xml_to_gracedb

```diff
@@ -158,7 +159,7 @@ for event in coinc_table:
             coinc_inspiral_table_curr.append(coinc_insp)
 
     if args.min_ifar is not None and \
-            coinc_inspiral_table_curr[0].combined_far > 1./args.min_ifar/lal.YRJUL_SI:
+            coinc_inspiral_table_curr[0].combined_far > conv.sec_to_year(1. / args.min_ifar):
```

Comment:
This is a bit tricky .. you are using the sec_to_year function on a _FAR_ (1/time) number, which makes it a factor pi*1e7 smaller, but it is rather difficult to parse/check as the function is written to convert times. 
Easier to deal with this in the same way as the previous ifar conversion :
```suggestion
            conv.sec_to_year(1. / coinc_inspiral_table_curr[0].combined_far) < args.min_ifar:  # event IFAR is smaller than the minimum
```